src/rest/resources/Emojis.ts

```diff
@@ -0,0 +1,50 @@
+import { RESTDeleteAPIGuildEmojiResult, RESTGetAPIGuildEmojiResult, RESTGetAPIGuildEmojisResult, RESTPatchAPIGuildEmojiJSONBody, RESTPatchAPIGuildEmojiResult, RESTPostAPIGuildEmojiJSONBody, RESTPostAPIGuildEmojiResult, Snowflake } from 'discord-api-types'
+import { RestManager } from '../Manager'
+
+/**
+ * Emojis resource
+ */
+export class EmojisResource {
+  constructor (private readonly rest: RestManager) {}
+
+  /**
+   * Gets all emojis in a guild, or a single emoji from a guild
+   * @param guildId ID of the emoji's guild
+   * @param emojiId The ID of the emoji
+   */
+  async get (guildId: Snowflake, emojiId?: Snowflake): Promise<RESTGetAPIGuildEmojisResult | RESTGetAPIGuildEmojiResult> {
```

Comment:
Either change this to a generic type where it returns the Emoji**s**Result if emojiId is present, or make a getMany() method. for typings sake.

Reply:
Done f4ace38